src/components/CreateFeeds.js

```diff
@@ -0,0 +1,100 @@
+import { useState } from 'react';
+import { Form, InnerForm } from './Styles'
+import { URL } from './Feeds';
+import Axios from 'axios';
+
+
+const CreateFeeds = () => {
+    const [details, setDetails] = useState({ author: "", title: "", content: "" })
+    const [loading, setLoading] = useState(false)
+
+
+    //function to set details
+    const setFeedDetails = (e) => {
+        setDetails(state => ({ ...state, [e.target.name]: e.target.value }))
+    }
+
+
+    //function to handle submit
+    const handleSubmit = async (e) => {
+        e.preventDefault();
+        setLoading(true)
+        try {
+            const result = await Axios.post(URL,
+                details
+            )
+            if (result?.status === 200) {
+                setLoading(false)
+            }
```

Comment:
A POST response's status is usually 201.

Do you actually need to check if `status === 2**`?

Reply:
yea but the backend guy switched it.

plus I just used to do the loading stuff.